packages/devtools/src/constants.ts

```diff
@@ -7,6 +7,7 @@ export const ETHERLIME_ACCOUNTS: Account[] = [
   {privateKey: '0xf473040b1a83739a9c7cc1f5719fab0f5bf178f83314d98557c58aae1910e03a'},
   {privateKey: '0x823d590ed2cb5e8493bb0efc834771c1cde36f9fc49b9fe3620ebd0754ad6ea2'},
   {privateKey: '0xd6d710943471e4c37ceb787857e7a2b41ca57f9cb4307ee9a9b21436a8e709c3'},
+  // Hub uses this (index 6)
```

Comment:
In theory, the `devtools` package could be used independent of the other packages. In which case, this comment might be a bit confusing. That being said, I am ok with having the comment